src/Raven.Server/Web/System/Processors/OngoingTasks/OngoingTasksHandlerProcessorForGetOngoingTasks.cs

```diff
@@ -0,0 +1,484 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using JetBrains.Annotations;
+using Raven.Client.Documents.Operations.Backups;
+using Raven.Client.Documents.Operations.ConnectionStrings;
+using Raven.Client.Documents.Operations.ETL;
+using Raven.Client.Documents.Operations.ETL.SQL;
+using Raven.Client.Documents.Operations.OngoingTasks;
+using Raven.Client.Documents.Operations.Replication;
+using Raven.Client.Documents.Subscriptions;
+using Raven.Client.Http;
+using Raven.Client.Json.Serialization;
+using Raven.Client.ServerWide;
+using Raven.Client.ServerWide.Operations;
+using Raven.Server.Documents;
+using Raven.Server.Documents.ETL;
+using Raven.Server.Documents.ETL.Providers.Raven;
+using Raven.Server.Documents.PeriodicBackup;
+using Raven.Server.Documents.Replication;
+using Raven.Server.Documents.Replication.Incoming;
+using Raven.Server.Documents.Replication.Outgoing;
+using Raven.Server.ServerWide;
+using Raven.Server.ServerWide.Context;
+using Sparrow.Json;
+
+namespace Raven.Server.Web.System.Processors.OngoingTasks;
+
+internal class OngoingTasksHandlerProcessorForGetOngoingTasks : AbstractOngoingTasksHandlerProcessorForGetOngoingTasks<DocumentsOperationContext>
+{
+    [NotNull]
+    private readonly DatabaseRequestHandler _requestHandler;
+    private readonly DocumentDatabase _database;
+    private readonly ServerStore _server;
+
+    public OngoingTasksHandlerProcessorForGetOngoingTasks([NotNull] DatabaseRequestHandler requestHandler, [NotNull] JsonContextPoolBase<DocumentsOperationContext> contextPool) : base(requestHandler, contextPool)
```

Comment:
can probably use the pool from requestHandler.ContextPool

Reply:
done